### ISSUE TITLE
Captcha API [REQUEST]

### ISSUE THREAD

**Issue by MagicalCat1172** [KSoft-Si/issues#1]
The problem is that whenever I tried to use a captcha API it just doesn't work for node.js.
I would like to have an api where if I do a post request to api.ksoft.si I can specify how long the captcha is and it will return a
{
     image: "link here",
     solution: "the solution to the captcha"
}

**Comment by BenjaminUrquhart** [comment]
Hey,

There shouldn’t be any captchas on any of the endpoints. Your request or IP was probably flagged by Cloudflare, so you got challenged. 

Gonna wait for Nani or ravy to handle this as they manage the Cloudflare stuff.

**Comment by MagicalCat1172** [comment]
No, it doesn't ask me for captcha. I want a new feature that can generate captchas.

**Comment by BenjaminUrquhart** [comment]
Oh my bad.

**Comment by PrincessRavy** [comment]
As of right now, all responses from our API are pure JSON data, not generated images. We'll look into this, but I cannot give you an ETA as it would be far future.
